package.json

```diff
@@ -47,10 +47,10 @@
     "vinyl-paths": "^1.0.0"
   },
   "scripts": {
-    "build": "gulp build-js && gulp build-sass",
+    "build": "gulp build-js && npm run build-legal && gulp build-sass",
```

Comment:
This seems like a circular/recursive "reference" to me if you will, is it okay to use `npm run ...` to in turn execute another `npm run ...`?